### PR TITLE
add: `joplin-desktop-app`

### PR DESCRIPTION
**Pull request by lfromanini**

### CODE REVIEW

**Comment by lfromanini** [comment]
This PR solves issue [#4983](https://github.com/pacstall/pacstall-programs/issues/4983).

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:15]:
```suggestion
```
The below line takes care of making it executable.

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:16]:
```suggestion
  sudo install -Dm755 "Joplin-${pkgver}.AppImage" "${pkgdir}/usr/bin/${gives}"
```

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (vigress8) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:24]:
```suggestion
    Exec=joplin-desktop %u
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

**Comment by lfromanini** [comment]
Done, @vigress8 .

**Review by vigress8 — APPROVED** [review]

**Comment by lfromanini** [comment]
@vigress8 , sorry, can you approve again after merging branch with master? Also, please, feel free to merge.

**Comment by vigress8** [comment]
@Elsie19 ?

**Review by vigress8 — APPROVED** [review]

**Comment by Elsie19** [comment]
Wait.

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:3]: Single quotes.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:4]:
```suggestion
pkgdesc="Joplin - the secure note taking and to-do app with synchronisation capabilities for Windows, macOS, Linux, Android and iOS"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (Elsie19) [inline packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: pkgver should be higher up.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/joplin-desktop-app/joplin-desktop-app.pacscript
Comment (lfromanini) [reply packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: No, 2.13.15 is the stable one. The higher ones are pre-release.

Comment (Elsie19) [reply packages/joplin-desktop-app/joplin-desktop-app.pacscript:8]: I meant the line should physically be higher up 😆

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**
